presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1142,7 +1145,22 @@ public SystemSessionProperties(
                         EXCEEDED_MEMORY_LIMIT_HEAP_DUMP_FILE_DIRECTORY,
                         "Directory to which heap snapshot will be dumped, if heap_dump_on_exceeded_memory_limit_enabled",
                         System.getProperty("java.io.tmpdir"),   // This is intended to be used for debugging purposes only and thus we does not need an associated config property
-                        true));
+                        true),
+                booleanProperty(
+                        KEY_BASED_SAMPLING_ENABLED,
+                        "Key based sampling of tables enabled",
```

Comment:
This is still not really a description of what this feature does. If you think there's no way to describe it clearly here, add proper documentation of this feature and refer it so people can learn more about it.

Reply:
Yeah - where do I add it? Add a chapter to the user doc?

Reply:
Actually the PR message has pretty much the documentation.